cmake/modules/shields.cmake

```diff
@@ -71,6 +71,10 @@ foreach(root ${BOARD_ROOT})
         continue()
       endif()
 
+      if(BOARD_DIR AND NOT (${root} STREQUAL ${ZEPHYR_BASE}))
```

Comment:
Some funky indentation here.

Reply:
Looks fine on my side. It's indented to the for loop nest level and then the inside of the if block is indented properly. Am I missing something obvious here? 🤔

Reply:
Nope...GH mobile rendering things poorly. All good actually.